problem-16.c

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdbool.h>
+// Basic calculator program that performs addition, subtraction, multiplication and division
+
+int getSum(int firstNum, int secondNum);
+int getDifference(int firstNum, int secondNum);
+int getProduct(int firstNum, int secondNum);
+int getQuotient(int firstNum, int secondNum);
+
+int main()
+{
+    int firstNum, secondNum, user_choice;

```

Comment:
- `user_choice` is in snake_case while others are in camelCase .... any particular reason?
- Better to declare it near the code where it's needed.